packages/matter.js/src/cluster/client/ClusterClient.ts

```diff
@@ -65,7 +55,7 @@ export function ClusterClient<F extends BitSchema, A extends Attributes, C exten
                     return await (attributes as any)[attributeName].get(isFabricFiltered, alwaysRequestFromRemote);
                 },
                 StatusResponseError,
-                e => {
+                (e: StatusResponseError) => {
```

Comment:
why you needed to annotate type? Should be automatically determined?

Reply:
I'm not sure, I get "Parameter 'e' implicitly has an any type" without the annotation.  Didn't look closely since plan is to kill tryCatchAsync